painttiming.bs

```diff
@@ -188,7 +188,7 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} of a [=global object=], where its [=responsible browsing context=] is [=paint-timing eligible=].
+NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} of a [=global object=]'s [=environment settings object=], whose [=responsible browsing context=] is [=paint-timing eligible=].
```

Comment:
```suggestion
NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} of a [=global object=] whose [=environment settings object=]'s [=responsible browsing context=] is [=paint-timing eligible=].
```
The point is that `responsible browsing context` is defined for an `environment settings object`.